src/components/shared/chart/Canvas.js

```diff
@@ -33,6 +33,8 @@ type Props<Item> = {|
 
   +onMouseMove?: (e: { nativeEvent: MouseEvent }) => mixed,
   +onMouseLeave?: (e: { nativeEvent: MouseEvent }) => mixed,
+  // Defaults to true. Set to false if the chart should not persist the tooltips.
+  +stickyTooltips?: boolean,
```

Comment:
nit: because it's used only in one graph, what do you think of defaulting to false instead of true? 
I would find the API clearer: instead of overriding a default, a user of the shared canvas would enable a feature.

Reply:
Hmm yeah, I can do that. My initial thought was that in the future if we add another chart I would like it to be sticky by default and if we choose to not make it, it would be better if it was a conscious choice. But currently it's only useful for markers. (actually there is a js tracer chart that uses too but I don't think we care about it)